data/src/main/java/thuy/flickr/PhotoRepositoryImpl.kt

```diff
@@ -46,4 +46,59 @@ internal class PhotoRepositoryImpl internal constructor(
           .doOnError { Timber.e(it, "Error getting photos") }
           .onErrorReturn { Failure(it) }
           .startWith(Busy())
+
+  private fun a(): Int {
+    return 0;
+  }
+
+  private fun b(): Int {
+    return 0;
+  }
+
+  private fun c(): Int {
+    return 0;
+  }
+
+  private fun d(): Int {
+    return 0;
+  }
+
+  private fun e(): Int {
+    return 0;
+  }
+
+  private fun f(): Int {
+    return 0;
+  }
+
+  private fun g(): Int {
+    return 0;
+  }
+
+  private fun h(): Int {
+    return 0;
+  }
+
+  private fun i(): Int {
+    return 0;
+  }
+
+  private fun j(): Int {
+    return 0;
+  }
+
+  private fun k() {
```

Comment:
<!--
  0 failure: 
  1 warning:  EmptyFunctionBloc...
  
  
  DangerID: danger-id-Danger;
  File: data/src/main/java/thuy/flickr/PhotoRepositoryImpl.kt;
  Line: 90;
-->

- :warning: EmptyFunctionBlock: This empty block of code can be removed.